src/mono/mono/utils/mono-logger-internals.h

```diff
@@ -96,7 +96,7 @@ mono_trace (GLogLevelFlags level, MonoTraceMask mask, const char *format, ...)
 #define mono_trace_warning(...) (mono_trace (G_LOG_LEVEL_WARNING, __VA_ARGS__))
 #define mono_trace_message(...) (mono_trace (G_LOG_LEVEL_MESSAGE, __VA_ARGS__))
 
-#if defined (HOST_ANDROID) || (defined (TARGET_IOS) && defined (TARGET_IOS))
+#if defined (HOST_ANDROID) || defined (TARGET_IOS) || defined (TARGET_TVOS)
```

Comment:
this may look weird at first but I traced it all the way back to https://github.com/mono/mono/commit/f31b8de0b9c5603a5725671f142b88a9dfc92e2d so the original intention probably doesn't matter anymore.